src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -171,6 +199,23 @@ public static class PermutationFeatureImportanceExtensions
                 auprc: a.AreaUnderPrecisionRecallCurve - b.AreaUnderPrecisionRecallCurve);
         }
 
+        private static CalibratedBinaryClassificationMetrics CalibratedBinaryClassifierDelta(
+            CalibratedBinaryClassificationMetrics a, CalibratedBinaryClassificationMetrics b)
```

Comment:
If I open this PR in Visual Studio I can see that this new method has 0 references, is this intended?

Particularly, notice that the `BinaryClassifierDelta ` (which already existed) gets a reference in `PermutationFeatureImportance<TModel>` weather `isCalibratedModel` is true or not.

Further more, I quickly ran another existing test for PFI for Binary Classification, with a calibrated model, and it still called the BinaryClassifierDelta instead of the new CalibratedBinaryClassifierDelta. So I wouldn't know if this is intended.

Reply:
I have removed it (I added it initially before we decided not to add calibrated metrics to PFI).

---
In reply to: [359531828](https://github.com/dotnet/machinelearning/pull/4587#discussion_r359531828) [](ancestors = 359531828)